harness/harness-core-ui
# fix: [CI-4023]: fetch correct summary and preselect node in tests tab

##### Summary:



##### Jira Links:



##### Screenshots:


![image](https://user-images.githubusercontent.com/43422351/166329626-0888f2ed-c1e2-4b2c-bd74-70debd8c1c00.png)

![image](https://user-images.githubusercontent.com/43422351/166329668-e9f2c662-8121-4e45-8af4-60cab8b421f9.png)

Preselected when moving to Test tab
![image](https://user-images.githubusercontent.com/43422351/166329716-0ca4a881-d767-455b-98b8-fafd00caf82b.png)

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`
